packages/terra-demographics-banner/tests/jest/DemographicsBannerDisplay.test.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import DemographicsBannerDisplay from '../../src/DemographicsBannerDisplay';
+
+it('renders a blank banner properly', () => {
+  const banner = shallow(<DemographicsBannerDisplay />);
+  expect(banner).toMatchSnapshot();
+});
+
+it('renders a banner that has basic information', () => {
+  const banner = shallow(
+    <DemographicsBannerDisplay
+      age="24"
+      dateOfBirth="March 30, 2017"
+      gender="F"
+      personName="Terra Demographics Banner"
+    />,
+  );
+
+  expect(banner).toMatchSnapshot();
+});
+
+it('renders the banner that contains additional information', () => {
+  const banner = shallow(
+    <DemographicsBannerDisplay
+      applicationContent={<span className="risk-score">5%</span>}
+      age="25 Years"
+      applicationRows={[{ Address: '1501 Walnut St. Apt #123', City: 'Kansas City MO' }, { ZIP: 64108 }]}
+      dateOfBirth="May 9, 1993"
+      gender="Male"
+      identifiers={{ MRN: 12343, REA: '3JSDA' }}
+      photo={<img alt="My Cat" src="" />}
```

Comment:
It's been requested to use the terra image component instead of a standard img tag. Could we perhaps add that change in the examples, as it's not too large?